.github/workflows/e2e-test.yml

```diff
@@ -72,6 +72,7 @@ jobs:
       - name: Run Greenfield-SPs
         run: |
           bash ./deployment/localup/localup.sh --generate $GITHUB_WORKSPACE/greenfield/sp.json ${{ env.MYSQL_USER }} ${{ env.MYSQL_PASSWORD }} ${{ env.MYSQL_IP }}:${{ env.MYSQL_PORT }}
+          bash ./deployment/localup/localup.sh --gene2e $GITHUB_WORKSPACE/greenfield/sp.json
```

Comment:
This cmd is only used in the integration test, why put it here?

Reply:
This cmd belongs to SP. You said it should be moved to `Run Greenfield E2E Test` job?

Reply:
Yes, because Run-SP does not depend on it.

Reply:
ok